src/_locales/en_US/messages.json

```diff
@@ -359,6 +359,22 @@
         "message": "Privacy Policy",
         "description": "Shown at the bottom of the intro page."
     },
+    "intro_easy_to_manage": {
+        "message": "Easy to manage",
+        "description": "Intro page paragraph heading on Chrome."
+    },
+    "intro_easy_to_manage_paragraph": {
+        "message": "Pin this extension to your toolbar for easy access. The Privacy Badger icon will show you how many trackers it blocks on each page. Click on the icon to manage your privacy settings.",
+        "description": "Intro page paragraph on Chrome."
+    },
+    "intro_pin_action": {
+        "message": "Pin Privacy Badger to your toolbar",
+        "description": "Shown on arrow pointing at Chrome extension icon"
```

Comment:
The translator probably wants to know how to find this. Also, no longer an arrow. "Part of a Chrome-only intro page overlay"?